acl_test.go

```diff
@@ -37,8 +37,7 @@ var (
 	unusedgroup  = "unused"
 	devgroup     = "dev"
 
-	dgraphAddress = "127.0.0.1:9180"
```

Comment:
we don't use this anymore?

Reply:
I guess. I could use it though, let me check

Reply:
not using this, but added a constant for the address